src/libraries/maintainers.ts

```diff
@@ -0,0 +1,266 @@
+      'start',
+      'router',
+      'query',
+      'table',
+      'form',
+      'virtual',
+      'ranger',
+      'store',
+      'react-charts',
+    ],
+    frameworkExpertise: ['react', 'solid'],
+    specialties: ['Architecture', 'Core API', 'Documentation'],
+  },
+  {
+    name: 'Dominik Dorfmeister',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/tkdodo.png',
+    github: 'tkdodo',
+    maintainerOf: ['query'],
+    contributorOf: ['router'],
+    frameworkExpertise: ['react'],
+    specialties: ['Core API', 'TypeScript', 'Documentation'],
+  },
+  {
+    name: 'Corbin Crutchley',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/crutchcorn.png',
+    github: 'crutchcorn',
+    creatorOf: ['form'],
+    maintainerOf: ['store'],
+    frameworkExpertise: ['react', 'solid', 'vue', 'angular'],
+    specialties: ['Forms', 'Validation', 'State Management'],
+  },
+  {
+    name: 'Manual Schiller',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/schiller-manuel.png',
+    github: 'schiller-manuel',
+    maintainerOf: ['start', 'router'],
+    frameworkExpertise: ['react'],
+    specialties: [],
+  },
+  {
+    name: 'Kevin Van Cott',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/kevinvandy.png',
+    github: 'kevinvandy',
+    creatorOf: ['pacer'],
+    maintainerOf: ['table'],
+    contributorOf: ['virtual'],
+    consultantOf: ['query'],
+    frameworkExpertise: ['react', 'solid', 'svelte'],
+    specialties: ['Tables', 'Data Grids', 'Dashboards'],
+  },
+  {
+    name: 'Sean Cassiere',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/seancassiere.png',
+    github: 'seancassiere',
+    maintainerOf: ['start', 'router'],
+    frameworkExpertise: ['react'],
+    specialties: [],
+  },
+  {
+    name: 'Chris Horobin',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/chorobin.png',
+    github: 'chorobin',
+    maintainerOf: ['router'],
+    frameworkExpertise: ['react'],
+    specialties: ['TypeScript'],
+  },
+  {
+    name: 'Damian Pieczynski',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/piecyk.png',
+    github: 'piecyk',
+    maintainerOf: ['virtual'],
+    frameworkExpertise: ['react'],
+    specialties: ['Virtualization', 'Performance'],
+  },
+  {
+    name: 'Jack Herrington',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/jherr.png',
+    github: 'jherr',
+    creatorOf: ['create-ts-router-app'],
+    frameworkExpertise: ['react'],
+    specialties: ['Templates'],
+  },
+  {
+    name: 'Kyle Mathews',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/KyleAMathews.png',
+    github: 'KyleAMathews',
+    creatorOf: ['create-ts-router-app'],
+    frameworkExpertise: ['react'],
+    specialties: ['Sync Engines'],
+  },
+  {
+    name: 'Lachlan Collins',
+    isCoreMaintainer: true,
+    avatar: 'https://github.com/lachlancollins.png',
+    github: 'lachlancollins',
+    maintainerOf: ['config'],
```

Comment:
@lachlancollins also maintains svelte-query

```suggestion
    maintainerOf: ['config', 'query'],
```